### ISSUE TITLE
Enable exclusions in the sssd-session-recording configuration

### ISSUE THREAD

**Issue by AndreasDavour** [Scribery/tlog#273]

**Comment by justin-stephenson** [comment]
@mzidek-rh @pbrezina Could you help me understand if the SSSD configuration interface would allow this type of exclusion or not?

**Comment by pbrezina** [comment]
Sure we could allow something like that in the configuration file. Is this something that tlog already supports and SSSD just does not allow to configure it?

**Comment by justin-stephenson** [comment]
@pbrezina Nothing is needed to enable this functionality on the tlog side AFAIK, only SSSD.

SSSD [session_recording] configuration section is used to override the shell as SESSION_RECORDING_SHELL used by NSS. See https://github.com/SSSD/sssd/pull/136/commits/b0cea9b316c9c6bc17b080be3d544fc07a2355f4 and https://github.com/SSSD/sssd/pull/136

**Comment by AndreasDavour** [comment]
So should this be an issue for https://github.com/SSSD/sssd instead?

**Comment by justin-stephenson** [comment]
@AndreasDavour yes, please file an issue there - most likely I will be the one implementing it still.

**Comment by AndreasDavour** [comment]
Thanks Justin. I will.

**Comment by AndreasDavour** [comment]
@justin-stephenson I created https://pagure.io/SSSD/sssd/issue/4128

Thanks.

**Comment by AndreasDavour** [comment]
Doesn't look like much is happening with the issue over there, right now.
@justin-stephenson do you want to keep this issue open for reference, or should I close it?

**Comment by justin-stephenson** [comment]
@AndreasDavour it is up to you, but I am fine to keep this open until the fix is done in SSSD.

**Comment by AndreasDavour** [comment]
Let's keep this open for visibility then.
